packages/hawtio/jest.config.ts

```diff
@@ -1,41 +1,59 @@
 import type { Config } from 'jest'
-import * as path from 'path'
 
 const config: Config = {
+  // https://jestjs.io/docs/ecmascript-modules
```

Comment:
Should this use multiline comment /* */ style instead?

Reply:
it was easier for me to do `ctrl+/`...
But sure, let's change it ;)